kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/resourcefulbees/bees.js

```diff
@@ -0,0 +1,5 @@
+onEvent('server.datapack.high_priority', (event) => {
+    bees.forEach((bee) => {
+        event.addJson(`resourcefulbees:loot_tables/entities/${bee}_bee.json`, {type: "minecraft:entity", pools: [{rolls: 1, entries: [{type: "minecraft:item", name: `kubejs:${bee}_larva`}]}]});
```

Comment:
I feel like the json should be formatted a bit here :P 
I highly recommend that you install Prettier if you're using VSCode :D

Reply:
Should be fixed!